src/components/net/linux-net.c

```diff
@@ -343,8 +348,11 @@ _net_init_component( int cidx  )
     _net_native_events = (NET_native_event_entry_t*)
         papi_malloc(sizeof(NET_native_event_entry_t) * num_events);
     do {
-        strncpy(_net_native_events[i].name, t->name, PAPI_MAX_STR_LEN);
-        strncpy(_net_native_events[i].description, t->description, PAPI_MAX_STR_LEN);
+        int retlen;
+        retlen = snprintf(_net_native_events[i].name, PAPI_MAX_STR_LEN, "%s", t->name);
+        if (retlen <= 0 || retlen >= PAPI_MAX_STR_LEN) HANDLE_STRING_ERROR;
```

Comment:
Probably, exiting the user program just because of a string truncation is not a good idea. We could replace HANDLE_STRING_ERROR to just log a message using, e.g., `SUDBG`. I know other part of the component do that but we should fix them too. Maybe we can address this problem in another PR though.

Reply:
I was thinking that exiting on truncation might have been a bit more than desired for those places too.

For components just use the component name.  What is the naming conventions for things in other locations like src/papi*.[ch],  testlib, ctests, utils, validation_tests, etc.?

I can work revising the patches this week.

Reply:
I think it is safe to use the name of the directory for components (e.g., cuda, rocm, etc) and tests. For things like the PAPI framework, maybe, we can agree on some naming convention (e.g., framework). I think I used `memory` or `threads` in the past for modifying code that belongs to either of those subsystems. Generally, anything that can give a good hint of what part of PAPI is affected by the changes in the commit would be good for history readability.